### ISSUE TITLE
JS error when adding to cart in IE: Object doesn't support property or method 'jscomp_symbol_iterator0' OR 'Symbol(Symbol.iterator)'

### ISSUE THREAD

**Issue by sacr3dc0w** [bigcommerce/cornerstone#1372]
### Expected behavior

Add to cart with no JS errors in IE11.

### Actual behavior

Product is added to cart, but object error occurs. I didn't notice this until we installed tracking software noticed our IE11 traffic was throwing this error. These errors occur regardless if a product option set is used.

`Object doesn't support property or method 'jscomp_symbol_iterator0'`

**Stack**
```
TypeError: Object doesn't support property or method 'jscomp_symbol_iterator0'
at filterEmptyFilesFromForm (http://10.0.0.74:3000/stencil/00000000-0000-0000-0000-000000000001/dist/theme-bundle.main.js:1444:89)
at addProductToCart (http://10.0.0.74:3000/stencil/00000000-0000-0000-0000-000000000001/dist/theme-bundle.main.js:1659:9)
at Anonymous function (http://10.0.0.74:3000/stencil/00000000-0000-0000-0000-000000000001/dist/theme-bundle.main.js:1407:13)
at jQuery.event.dispatch (http://10.0.0.74:3000/stencil/00000000-0000-0000-0000-000000000001/dist/theme-bundle.main.js:19892:6)
at elemData.handle (http://10.0.0.74:3000/stencil/00000000-0000-0000-0000-000000000001/dist/theme-bundle.main.js:19700:5)
```

`Object doesn't support property or method 'Symbol(Symbol.iterator)'`

**Stack**
```
TypeError: Object doesn't support property or method 'Symbol(Symbol.iterator)_7.fw2y2bsv01l'
   at filterEmptyFilesFromForm (http://10.0.0.74:3000/stencil/00000000-0000-0000-0000-000000000001/dist/theme-bundle.main.js:1444:89)
   at addProductToCart (http://10.0.0.74:3000/stencil/00000000-0000-0000-0000-000000000001/dist/theme-bundle.main.js:1659:9)
   at Anonymous function (http://10.0.0.74:3000/stencil/00000000-0000-0000-0000-000000000001/dist/theme-bundle.main.js:1407:13)
   at jQuery.event.dispatch (http://10.0.0.74:3000/stencil/00000000-0000-0000-0000-000000000001/dist/theme-bundle.main.js:19892:6)
   at elemData.handle (http://10.0.0.74:3000/stencil/00000000-0000-0000-0000-000000000001/dist/theme-bundle.main.js:19700:5)
```

### Steps to reproduce behavior

Install Cornerstone, Boot up in IE, go to a product and add it to your cart.

<img width="1552" alt="screen shot 2018-10-22 at 6 31 29 pm" src="https://user-images.githubusercontent.com/5056945/47329398-63225e00-d629-11e8-959e-735cb363b7f2.png">
<img width="1552" alt="screen shot 2018-10-22 at 6 36 47 pm" src="https://user-images.githubusercontent.com/5056945/47329423-7df4d280-d629-11e8-9034-4c73a3d7f799.png">

**Comment by karen-white** [comment]
Thanks for flagging this @sacr3dc0w ! I've opened an engineering ticket to address it.

**Comment by PeteyRev** [comment]
@sacr3dc0w Had this issue recently, it seems to be an ie11 polyfill issue, updated the weback babel preset-env configuration to target ie11: 
```
            presets: [
              ['@babel/preset-env', {
                loose: true, // Enable "loose" transformations for any plugins in this preset that allow them
                modules: false, // Don't transform modules; needed for tree-shaking
                useBuiltIns: 'usage', // Tree-shake babel-polyfill
                targets: '> 1%, last 2 versions, Firefox ESR, ie >= 11',
              }],
            ]
```

**Comment by sacr3dc0w** [comment]
Resolved by #1762
